server/__init__.py

```diff
@@ -1,19 +1,18 @@
 from flask import Flask
+import os
 from flask_sqlalchemy import SQLAlchemy
 from flask_bcrypt import Bcrypt
 from flask_login import LoginManager, login_manager
 
+SECRET_KEY = os.getenv("SECRET_KEY")
+
```

Comment:
Ok, since when I was testing here I've got a cryptic error message ("TypeError: str object is expected" when sending a request to `/users/login`) because SECRET_KEY wasn't set on my environment, I would recommend the following:

```suggestion

if SECRET_KEY is None:
    sys.exit(dedent("""
        SECRET_KEY is not set. Hint: create a .env file containing:

        SECRET_KEY=<randomly generated key>

        To generate a key, see https://flask.palletsprojects.com/en/latest/config/#SECRET_KEY.
    """).strip())

```

This requires importing these at the start of the file:

```python
import sys
from textwrap import dedent
```

Reply:
Does it mean a new `SECRET_KEY` would be created whenever any new user forks this project and starts the server?

Reply:
![Screenshot from 2021-07-30 11-14-54](https://user-images.githubusercontent.com/56754747/127606524-8dd4c97f-d505-4304-bfa5-b48e0078a440.png)

@JPTIZ, I'm facing a strange issue.
I'm getting the value of `SECRET_KEY` as `None` even if I've set its value in `.env` file. 

If I comment out that `if` block where it checks the value of `SECRET_KEY` then server starts correctly